Resources/Textures/Objects/Weapons/Guns/HMGs/minigun.rsi/meta.json

```diff
@@ -7,45 +7,12 @@
     "y": 32
   },
   "states": [
-    {
-      "name": "base",
```

Comment:
Is this ok?

Reply:
ye. Though I might add a visualizer for this so I will split out an icon.